client/components/editor/teaching-elements/Assessment/ImageMultipleChoice.vue

```diff
@@ -52,6 +52,15 @@ const formAlert = (message) => {
   };
 };
 
+const getImageDataUrl = (base64String, mimeType) => `data:${mimeType};base64,${base64String}`;
+
+const extractImageDimensions = (base64String, mimeType, cb) => {
+  let i = document.createElement('img');
+  // eslint-disable-next-line standard/no-callback-literal
+  i.onload = () => cb({ width: i.width, height: i.height });
+  i.src = getImageDataUrl(base64String, mimeType);
+};
```

Comment:
Helper function is a _function_ not arrow function ❕ 
Arrow functions are ok if they are small enough, preferably one-liners...

Also, why did you make this callback based instead of returning Promise?